decidim-proposals/app/views/decidim/proposals/admin/proposals/_proposal-tr.html.erb

```diff
@@ -1,4 +1,7 @@
-<tr data-id="<%= proposal.id %>" <%= "data-published-state=false" if proposal.answered? && !proposal.published_state? %>>
+<tr class="<%= "admin-highlighted" if proposal.published_at < Decidim::Proposals
+  .process_group_highlighted_proposals_overdue
+  .days.ago && !proposal.answered? %>" data-id="<%= proposal.id %>"
+    <%= "data-published-state=false" if proposal.answered? && !proposal.published_state? %>>
```

Comment:
```suggestion
<tr class="<%= "admin-highlighted" if proposal.overdue? %>" data-id="<%= proposal.id %>"
    <%= "data-published-state=false" if proposal.answered? && !proposal.published_state? %>>
```

We should put all this code in a helper and call that method instead .
You can add the additional method `def overdue?` to the Proposal model